src/js/views/Projects/HelpDialog.jsx

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types'
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { Modal } from '../../components'
+import { Context } from '../../state'
+
+function HelpDialog({ onClose }) {
+  const { t } = useTranslation()
+  const [globalState] = useContext(Context)
+  let rowOffset = -1
+  return (
+    <Modal onClose={onClose}>
+      <Modal.Title>{t('projects.searchHelpTitle')}</Modal.Title>
+      <div>
+        <span
+          dangerouslySetInnerHTML={{
+            __html: t('projects.searchHelpDQL', {
+              interpolation: { escapeValue: false }
+            })
+          }}
+        />
+        <h1 className="my-4 font-bold">{t('projects.searchHelpFields')}</h1>
+        <ul className="list-disc list-inside max-h-36 ml-5 font-mono overflow-scroll">
+          {globalState.openSearch.fields.map((field) => {
+            rowOffset += 1
+            return <li key={`field-` + rowOffset}>{field.name}</li>
```

Comment:
Is it possible for the openSearch fields to change order or be filtered? [React recommends using a stable unique value instead of an index](https://reactjs.org/docs/lists-and-keys.html#keys). `field.name` may work?

Reply:
The chances of the values changing or be reordered during an individual session in using the app are incredibly small (<1%, would require admin privs and adding a AND new field, updating a project fact during the session for that session only)

Reply:
Turns out the issue was due to duplicate values coming in from the API.